### ISSUE TITLE
Custom goal ignoring Warmup filter

### ISSUE THREAD

**Issue by wintyre01** [hmxmilohax/rock-band-3-deluxe#563]
Decided to make my own custom goal, but running into an issue with getting it to only allow me to play a specific difficulty tier

From dx_goal_hooks.dta:
{push_back {find $syscfg campaign accomplishment_info accomplishments} ; Warmup Band
   (acc_dx_warmup_band
      (category acc_cat_band_dx)
      (accomplishment_type kAccomplishmentTypeSongFilterConditional)
      (metascore_value career_fans_easy)
      (launchable_playercount_min 2)
      (units_token unit_songs_plural)
      (units_token_singular unit_songs_singular)
      (num_songs 40)
      (conditions
         (stars 5))
   )
   (part_difficulty_sym band)
   (filter
      (kFilterDifficulty generic_tier1))
}

From dx_locale_updates.dta:
(acc_dx_warmup_band
   "\qIt's Exposure Bro\q")
(acc_dx_warmup_band_desc
   "Earn 5 <alt>*</alt> on 40 Warmup songs")

Basically I want this goal to only allow me to play Warmup songs (outside of RB3 included), but it's ignoring the filter I have set. I was planning to do this with the other tiers as well.

**Comment by wintyre01** [comment]
Found out what it was:

(part_difficulty_sym band)
(filter
(kFilterDifficulty generic_tier1))

This segment wasn't enclosed in the goal parameters
